lang/cs/Org.Apache.REEF.Examples.HelloREEF/HelloREEFYarn.cs

```diff
@@ -115,6 +114,17 @@ private void LogApplicationReport()
         }
 
         /// <summary>
+        /// Kill Job Application.
+        /// </summary>
+        /// <param name="appId"></param>
+        private void KillJobApplication(string appId)
```

Comment:
I called it after submitting the job in testing and removed call. I can leave the all and comment it out to show an example for the client.